cataloguing/additem.pl

```diff
@@ -679,6 +686,7 @@ sub get_common_fields_template {
                     # We count the item only if it was really added
                     # That way, all items are added, even if there was some already existing barcodes
                     # FIXME : Please note that there is a risk of infinite loop here if we never find a suitable barcode
+                    # NOT FIX but possibility to detect issue: ($total_iterations - $i) > $close_to_infinity counter on the beginning of loop.
                     $i++;
```

Comment:
@kohaputti this is incremented when real item added, so `i `will to towards `number_of_copies` goal anyway